src/main/kotlin/racingcar/domain/Race.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain
+
+class Race(private val cars: List<Car>) {
+    constructor(carCount: Int) : this(List(carCount) { Car() })
+
+    fun run() {
+        cars.forEach { it.move() }
+    }
+
+    val movingDistances
+        get() = this.cars.map { it.position }
+
+    val carsSize
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    val movingDistances: List<Int>
        get() = this.cars.map { it.position }

    val carsSize: Int
```